WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAccountOverdueView.swift

```diff
@@ -13,9 +15,28 @@ struct InPersonPaymentsStripeAccountOverdue: View {
             ),
             supportLink: true,
             learnMore: true,
-            analyticReason: analyticReason
+            analyticReason: analyticReason,
+            buttonViewModel: InPersonPaymentsOnboardingErrorButtonViewModel(text: Localization.primaryButtonTitle,
+                                                                            analyticReason: analyticReason,
+                                                                            action: {
+                                                                                presentedSetupURL = setupURL
+                                                                            }),
+            secondaryButtonViewModel: InPersonPaymentsOnboardingErrorButtonViewModel(text: Localization.secondaryButtonTitle,
+                                                                                     analyticReason: analyticReason,
+                                                                                     action: {
```

Comment:
We could directly pass the closure here:

`action: onRefresh)`

Reply:
Sounds good, changed here: https://github.com/woocommerce/woocommerce-ios/pull/10663/commits/a67df1abaf8f1f255340bd0e0112d5c03715c48a